CardScan/Classes/CreditCardUtils.swift

```diff
@@ -1,29 +1,43 @@
 import Foundation
 
+public enum CardNetwork: String {
```

Comment:
does this need to be public?

Reply:
its public so it can be used in `CardVerify`